invenio_accounts/ext.py

```diff
@@ -322,3 +318,39 @@ def make_session_permanent(self, app):
         @app.before_request
         def make_session_permanent():
             session.permanent = True
+
+
+def finalize_app(app):
+    """Finalize app."""
+    post_ext_init(app)
+    check_security_settings(app)
+
+
+def post_ext_init(app):
```

Comment:
```suggestion
def set_default_config(app):
```

nit: to be more descriptive, and maybe add a docstring as well